packages/auth/src/logger/service.ts

```diff
@@ -1,11 +0,0 @@
-import createLogger, { Logger as PinoLogger } from 'pino'
```

Comment:
Might still be relevant for https://github.com/interledger/rafiki/pull/2697/files#diff-9c3c6613a84f50df315ff190bc76ca450044920269a9467a6ad13e0bf10753c5

Reply:
I think it's not, actually. Just me covering all the bases but I don't plan on expanding its usage.